go/libkb/secret_store.go

```diff
@@ -173,6 +174,29 @@ func GetConfiguredAccountsFromProvisionedUsernames(m MetaContext, s SecretStoreA
 		configuredAccounts = append(configuredAccounts, account)
 	}
 
+	logins, err := getLoginTimes(m)
```

Comment:
can you rename this to `loginTimeMap` or something? sorting is slightly confusing to read as is

Reply:
ok laszlo